src/pages/Home/index.tsx

```diff
@@ -51,6 +51,19 @@ const Home = () => {
         icon="waving.svg"
         id="product"

```

Comment:
- **Optimization**  
  The `Contact` component is rendered multiple times with the same props. Consider storing the result in a variable to improve performance.